apps/weatherdashboard/manifest.yaml

```diff
@@ -0,0 +1,8 @@
+---
+id: weather-dashboard
+name: Weather Dashboard
```

Comment:
Would be great if the metadata could be updated to reflect that it's specifically weather stations from Ambient Weather that's supported.

Reply:
I updated the description to reference Ambient Weather specifically. I'm running into character limits on the name but will be happy to change it if you have anything specific